.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: Test
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  pytest:
+    runs-on: "ubuntu-latest"
+    name: Run tests
+    steps:
+      - uses: "actions/checkout@v2"
+      - uses: "actions/setup-python@v1"
+        with:
+          python-version: "3.8"
```

Comment:
Isn't HA on 3.9 now?  Even if it is on 3.8, let's do a matrix and test on both, since I expect them to move to 3.9 pretty quickly based on history.

Reply:
HA uses both 3.8 and 3.9, it makes sense to test both here too.  I updated the setup python action version too.